src/Ilios/CliBundle/Command/SyncAllUsersCommand.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace Ilios\CliBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Doctrine\ORM\EntityManager;
+
+use Ilios\CoreBundle\Entity\Manager\UserManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\AuthenticationManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\PendingUserUpdateManagerInterface;
+use Ilios\CoreBundle\Service\Directory;
+
+/**
+ * Sync a user with their directory information
+ *
+ * Class SyncUserCommand
+ * @package Ilios\CliBUndle\Command
+ */
+class SyncAllUsersCommand extends Command
+{
+    /**
+     * @var UserManagerInterface
+     */
+    protected $userManager;
+
+    /**
+     * @var AuthenticationManagerInterface
+     */
+    protected $authenticationManager;
+
+    /**
+     * @var PendingUserUpdateManagerInterface
+     */
+    protected $pendingUserUpdateManager;
+    
+    /**
+     * @var Directory
+     */
+    protected $directory;
+    
+    /**
+     * @var EntityManager
+     */
+    protected $em;
+    
+    public function __construct(
+        UserManagerInterface $userManager,
+        AuthenticationManagerInterface $authenticationManager,
+        PendingUserUpdateManagerInterface $pendingUserUpdateManager,
+        Directory $directory,
+        EntityManager $em
+    ) {
+        $this->userManager = $userManager;
+        $this->authenticationManager = $authenticationManager;
+        $this->pendingUserUpdateManager = $pendingUserUpdateManager;
+        $this->directory = $directory;
+        $this->em = $em;
+        
+        parent::__construct();
+    }
+    
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('ilios:directory:sync-users')
+            ->setDescription('Sync all users against the directory by their campus id.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln('<info>Starting User Sync Process</info>');
+        $this->userManager->resetExaminedFlagForAllUsers();
+        $this->pendingUserUpdateManager->removeAllPendingUserUpdates();
+        $campusIds = $this->userManager->getAllCampusIds(false, false);
+        $output->writeln(
+            '<info>Attempting to update the ' .
+            count($campusIds) .
+            ' enabled and non sync ignored users in the system.</info>'
+        );
+        $output->writeln('<info>Searching the directory for users.</info>');
+        $allUserRecords = $this->directory->findByCampusIds($campusIds);
+        
+        if (!$allUserRecords) {
+            $output->writeln('<error>[E] Unable to find any users in the directory</error>');
```

Comment:
I believe they will still come in handy for hunt and peck seeking out of specific issues/categories of issues.